addons/l10n_fr/data/account_tax_data.xml

```diff
@@ -162,7 +162,7 @@
                 'factor_percent': 100,
                 'repartition_type': 'tax',
                 'minus_report_line_ids': [ref('l10n_fr.tax_report_tva_coll_8_5')],
-                'account_id': ref('pcg_445713')
```

Comment:
Really necessary?  It is just a small difference between putting it on a specific or generic account

Reply:
Yes, that's it. Product Owners want to only keep the generic account and let people create the specific ones.